pkgs/by-name/ni/nixel/package.nix

```
@@ -2,22 +2,33 @@
   lib,
   rustPlatform,
   fetchFromGitHub,
+  nix-update-script,
+  clang_16,
   testers,
   nixel,
 }:
-
-rustPlatform.buildRustPackage rec {
+let
+  clang = clang_16;
+  rustPlatform' = rustPlatform // {
+    bindgenHook = rustPlatform.bindgenHook.override { inherit clang; };
+  };
+in
+rustPlatform'.buildRustPackage rec {
   pname = "nixel";
-  version = "4.1.0";
+  version = "5.1.1";
 
   src = fetchFromGitHub {
     owner = "kamadorueda";
     repo = pname;
     rev = version;
-    sha256 = "sha256-dQ3wzBTjteqk9rju+FMAO+ydimnGu24Y2DEDLX/P+1A=";
+    sha256 = "sha256-wIXZd8iujLqbgbV+abWmIfjDgQ8i6nyE5jTAs3OczXM=";
   };
 
-  cargoHash = "sha256-1OsHs0W3ji9Kgpv7nGY9XyGxJ4c0faN2VuFLsdwkgKY=";
+  cargoHash = "sha256-GJ2UO8MvujRxFjzdt2Tfrlfzv1jot9XCrmfBqTFVngI=";
+
+  nativeBuildInputs = [
+    rustPlatform'.bindgenHook # .override { clang = clang_16; }
```

```suggestion
    rustPlatform'.bindgenHook
```
This might be intended to indicate what exactly `rustPlatform'` _is_, but it also looks like a commented out alternative idea.

---

That's both of them xD